public/javascripts/SVLabel/src/SVLabel/navigation/MapService.js

```diff
@@ -587,6 +587,15 @@ function MapService (canvas, neighborhoodModel, uiMap, params) {
         return uiMap.pano.find('svg').parent();
     }
 
+    /**
+     * Get link layer of bottom links
+     * @returns {*}
+     */
+    function getBottomLinkLayer () {
+        var links = $(uiMap.pano.find("a")[6]).parent().parent().parent();
```

Comment:
Is there a way to identify those links at the bottom in a way that is more descriptive than going to the 7th "a" within `uiMap.pano`? I'd also probably prefer just returning the value over saving it in a variable and then returning it.